src/endpoints/createOffersEndpoint.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import OfferModel from '../models/OfferModel'
+import EndpointBuilder from '../EndpointBuilder'
+
+import ParamMissingError from '../errors/ParamMissingError'
+import type { JsonOfferPostType, JsonOfferType } from '../types'
+import Endpoint from '../Endpoint'
+
+export const OFFERS_ENDPOINT_NAME = 'extras'
+
+const createPostMap = (jsonPost: JsonOfferPostType): Map<string, string> => {
+  const map = new Map()
+  Object.keys(jsonPost).forEach(key => map.set(key, jsonPost[key]))
+  return map
+}
+
+type ParamsType = { city: ?string, language: ?string }
```

Comment:
I think this type is wrong, right? Should not be optional.

Reply:
Hmm...not sure about that on the native end, but on the web app city and language are always present